packages/cli/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@mermaidchart/cli",
+  "version": "0.1.0-alpha.0",
+  "description": "CLI for interacting with https://MermaidChart.com, the platform that makes collaborating with Mermaid diagrams easy",
+  "main": "index.js",
+  "bin": {
+    "mermaid-chart": "dist/cli.js"
+  },
+  "//1": "temporarily disable pushing until we finish progress on this",
+  "private": true,
+  "engines": {
+    "node": "^18.18.0 || ^20.0.0"
+  },
+  "scripts": {
+    "prepare": "tsc --build tsconfig.json",
+    "test": "vitest"
+  },
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mermaid-Chart/mermaid-chart-cli.git"
+  },
+  "keywords": [
+    "mermaid",
+    "mermaidchart",
+    "cli"
+  ],
+  "author": "Alois Klink <alois@mermaidchart.com> (https://github.com/aloisklink)",
+  "//2": "TODO: decide on a license once this code is finished",
+  "license": "UNLICENSED",
```

Comment:
@knsv, what do you want the license for this package to be?

MIT is the license in the root of the repo, so we could use that: https://github.com/Mermaid-Chart/plugins/blob/main/LICENSE